main/zoe/guide/contracts/constantProductAMM.md

```diff
@@ -2,7 +2,7 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/vpool-xyk-amm/multipoolMarketMaker.js) (Last updated: 2021-10-26)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/15322a1f65bb67e1865f8276600a76171696dc74/packages/zoe/src/contracts/vpool-xyk-amm/multipoolMarketMaker.js) (Last updated: Feb 17, 2022)
```

Comment:
@Chris-Hibbert multipoolMarketMaker.js was moved to the run-protocol package by https://github.com/Agoric/agoric-sdk/commit/a9895c41b57d5ab3543a213824e293ee304dbcaa. This update points to its state immediately before the move, but we could also follow it if we're willing to have a Zoe contracts page link out of the zoe package.

Reply:
@Chris-Hibbert Done, please verify that the link is correct.

Reply:
I think this is the link 
https://github.com/Agoric/agoric-sdk/blob/363115d47652c43bec8514f624524e70e30f5b8e/packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

Reply:
And is https://github.com/Agoric/agoric-sdk/blob/4e0aece631d8310c7ab8ef3f46fad8981f64d208/packages/run-protocol/src/vaultFactory/vaultFactory.js correct for the [Treasury contract page](https://github.com/Agoric/documentation/pull/642/files#diff-dd1d62f3cce7d3383e80a0c75a5ee4e38a5e843a708c9863ee44e8aea265d57c)?

Reply:
Yes.